scripts/shm_demo.sh

```diff
@@ -3,11 +3,22 @@
 session="demo"
 window="ros2 vision demo"
 
-ros2_ws="~/ade-home/ros2_rolling/"
-package="ros2_shm_vision_demo"
-#set some video path
-video="./video/apex_ai.mp4"
+ros2_ws="$HOME/code/ros/rolling"
+package="shm_vision_demo"
+iceoryx_path="$ros2_ws/install/iceoryx_posh/bin"
```

Comment:
@MatthiasKillat for the `shm_demo.sh`, you'll need to adjust the ros2_ws again, along with the `iceoryx` path as well. I don't know how to get around these hard-coded paths, but at least now `cmake` handles the installation of the yolov3.weights and copies it all to the `install` directory.